mat64/eigen.go

```diff
@@ -817,3 +820,49 @@ func (f EigenFactors) D() *Dense {
 	}
 	return dm
 }
+
+// MarshalBinary encodes the receiver into a binary form and returns the result.
+func (f EigenFactors) MarshalBinary() ([]byte, error) {
```

Comment:
I see the utility of having marshalers for Dense, but do we really need the code for these other values (not just this, but all the supplemental structs)? I think we should keep the code minimal until there's a demonstrated need to marshal a particular struct. This is especially true as some of these structs may change, see for example Issue #79